tools/createSovereignGenesis/create-sovereign-genesis.ts

```diff
@@ -145,6 +167,52 @@ async function main() {
         outWETHAddress = wethObject.address;
     }
 
+    // check preMintAddress is an address
+    if(ethers.isAddress(createGenesisSovereignParams.preMintAddress) == false){
+        logger.error('preMintAddress: not a valid address');
+        throw new Error('preMintAddress: not a valid address');
+    }
+
+    // add preMint address & preMint value
+    logger.info('Add preMintAddress and preMintAmount');
```

Comment:
add some check that this address was not sued previously in the genesis?¿
Also could make sense to have a no premintaddress, maybe we can make a exception, in case of address 0 do not set this

Reply:
I will do the same for the `timelockAdmin` in fact.
Just in case the genesis-base admin is not wanted to be changed